core/src/main/java/org/apache/james/core/quota/QuotaComponent.java

```diff
@@ -23,6 +23,14 @@
 
 public class QuotaComponent extends StringWrapper {
 
+    public static final QuotaComponent MAILBOX = of("mailbox");
+    public static final QuotaComponent SIEVE = of("sieve");
+    public static final QuotaComponent JMAP_UPLOADS = of("jmapUploads");
+
+    public static QuotaComponent of(String value) {
```

Comment:
What's the point here?

Could just do `public static final QuotaComponent MAILBOX = new QuotaComponent("mailbox");` right?

Reply:
Just a shorter way to create new QuotaComponent. It does not matter which way I use, right?

Reply:
I find adding a whole static method just to pass from `new QuotaComponent` to `of` is adding more verbosity for... not much any kind of gain?

Reply:
Make the constructor private then?